src/security/test_security.py

```diff
@@ -3317,6 +3319,36 @@ def test_section_editor_production_no_or_bad_article_id(self):
         with self.assertRaises(PermissionDenied):
             decorated_func(request, **bad_kwargs)
 
+    def test_loading_keyword_page_fail(self):
+        func = Mock()
+        decorated_func = decorators.keyword_page_enabled(func)
+
+        request = self.prepare_request_with_user(
+            self.admin_user,
+            self.journal_one,
+            self.press,
+        )
+
+        self.assertIsInstance(decorated_func(request), HttpResponseRedirect)
+
+    def test_loading_keyword_page_success(self):
+        func = Mock()
+        decorated_func = decorators.keyword_page_enabled(func)
+
+        setting_handler.save_setting('general', 'keyword_list_page', self.journal_one, 'on')
```

Comment:
This alters the state of the test, it should be either reverted or mocked (if this test runs before `test_loading_keyword_page_fail` I think the test would fail)